src/XIVLauncher.Common.Unix/UnixGameRunner.cs

```diff
@@ -88,10 +100,18 @@ public LinuxGameRunner(LinuxStartupType startupType, string startupCommandLine,
         Log.Information("Starting game with: {Command} {Args}", helperProcess.StartInfo.FileName, helperProcess.StartInfo.ArgumentList);
 
         helperProcess.Start();
+        Int32 gameProcessId = 0;
         helperProcess.BeginErrorReadLine();
-
-        string pid = helperProcess.StandardOutput.ReadToEnd();
-
-        return int.Parse(pid);
+        while (gameProcessId == 0)
+        {
+            Thread.Sleep(50);
+            gameProcessId = compatibility.GetProcessId("ffxiv_dx11.exe");
```

Comment:
How does this handle multiboxing? On Windows, we had to change this as it would always select the first process' PID.

Reply:
It would have to keep a list of pids where injection has been attempted I guess in order to properly deal with multiboxing, shouldn't be too hard to fix

Reply:
Does winedbg not let you go from unix pid to windows pid?

Reply:
I tried, but couldn't find anything on it unfortunately

Reply:
fixed with cf93cac2fcc7261cca3e5c3edd530b847d74118c

Reply:
So I guess we can merge this for now as it doesn't break Windows multiboxing, and whenever we get to build the pid resolver for wine, we can fix it in xlcore